alter.sql

```diff
@@ -0,0 +1,24 @@
+alter table animals
+add column species varchar;
+
+begin;
+update animals;
+set species = 'unspecified';
+select * from animals;
+rollback;
+select * from animals;
+
+begin;
+update animals 
+set species = 'digimon' where name ~ '.*(mon)';
+commit;
+
+begin;
+delete from animals where date_of_birth>'2022-01-01';
+savepoint beforeWeightUpdate;
+update animals
+set weight_kg = weight_kg * -1;
+rollback to savepoint beforeWeightUpdate;
+update animals
+set weight_kg = weight_kg * -1 where weight_kg < 0;
+commit;
```

Comment:
- [ ] According to the project requirement, you should create `queries.sql` file to answer all the following questions in transactions, Please move all of these SQL commands to the `queries.sql` file rather than the `alter.sql`  file.

Reply:
I already have `queries.sql` file.
https://github.com/damdafayton/database-projects/pull/2/files#diff-aed7bfe763344ffc724771decccc3395cf564d6cd9a3ba18e26f70479c93ee0a